Add Initial yaml for CCC 1.2.6

jdebacker opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team][1] using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel][2] if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/ccc```) and found it was in an excellent condition.

jdebacker commented:
@conda-forge/staged-recipes This PR is ready for your review.  Thanks!

chusloj commented:
@conda-forge-admin, please ping team

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I was asked to ping @conda-forge/staged-recipes and so here I am doing that.

dopplershift commented on recipes/ccc/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+about:
+  home: https://github.com/PSLmodels/Cost-of-Capital-Calculator
+  license: CC-BY-1.0 # see SPDX identifiers
+  license_family: CC
```
Probably should be `Public-Domain`

jdebacker replied:
Updated, thanks.

dopplershift commented on recipes/ccc/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+
+about:
+  home: https://github.com/PSLmodels/Cost-of-Capital-Calculator
+  license: CC-BY-1.0 # see SPDX identifiers
```
`CC0-1.0` is what they claim in the license. See the [SPDX identifier](https://spdx.org/licenses/CC0-1.0.html).

jdebacker replied:
Updated.

dopplershift commented on recipes/ccc/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+    - taxcalc >=3.1.0
+    - bokeh >=1.4.0
+    - paramtools >=0.7.1
+    - setuptools
```
It looks like `setuptools` is only needed to run `setup.py`, so I *think* you can move this up under `host` instead.

jdebacker replied:
Good call - moved.

dopplershift commented on recipes/ccc/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+    - bokeh >=1.4.0
+    - paramtools >=0.7.1
+    - setuptools
+    - xlrd
```
Do you really need to depend on this? I don't see any imports of this in the code base on GitHub, but do see a commit saying "Remove xlrd dependency"

jdebacker replied:
Thanks for looking at this.  `xlrd` is not used in the package, but is used if one wants to recreate data contained in the package with scripts contained in repo.  Dropping this from the requirements here.

dopplershift commented:
Looks like cs2tc is not yet available from conda-forge. You'll need to add it, either in this PR or in another PR that gets merged first.

jdebacker commented:
@dopplershift Ok - I just removed `cs2tc` as a dependency.  That's only needed when running this package in a [web application](http://compute.studio/PSLmodels/Cost-of-Capital-Calculator/) and won't need to be in the conda-fogre package as I thought it would.

dopplershift commented:
Looks like the failure is because it can't find the license file. Any reason not to use the one you originally had set?

jdebacker commented:
@dopplershift No - that file was fine.  I just cleaned it up and moved it to a more obvious spot in the docs in [a PR today](https://github.com/PSLmodels/Cost-of-Capital-Calculator/pull/340) and didn't realize the path to the license file for the `meta.yaml` was pointing to the location in the tarball (and not the master branch of the repo).
